lesson_03/homework03_v2.py

```diff
@@ -0,0 +1,133 @@
+# task 01 == Розділіть змінну alice_in_wonderland так, щоб вона займала декілька фізичних лінії
+# task 02 == Знайдіть та екрануйте всі символи одинарної дужки у тексті
+# task 03 == Виведіть змінну alice_in_wonderland на друк
+
+alice_in_wonderland = ('"Would you tell me, please, which way I ought to go from here?"'
+                        '\n"That depends a good deal on where you want to get to," said the Cat.'
+                        '\n"I don\'t much care where ——" said Alice.'
+                        '\n"Then it doesn\'t matter which way you go," said the Cat.'
+                        '\n"—— so long as I get somewhere," Alice added as an explanation.'
+                        '\n"Oh, you\'re sure to do that," said the Cat, "if you only walk long enough."')
+
+print(alice_in_wonderland)
+
+
+separator = '- ' * 38
+print(separator)
+
+# task 04
+"""
+Площа Чорного моря становить 436 402 км2, а площа Азовського
+моря становить 37 800 км2. Яку площу займають Чорне та Азов-
+ське моря разом?
+"""
+
+print(f'Area of the Black Sea = {436402} km2\n'
+    f'Area of the Azov Sea = {37800} km2\n'
+    f'Area of the Black Sea and the Sea of Azov together = {436402 + 37800} km2')
+
+print(separator)
+
+# task 05
+"""
+Мережа супермаркетів має 3 склади, де всього розміщено
+375 291 товар. На першому та другому складах перебуває
+250 449 товарів. На другому та третьому – 222 950 товарів.
+Знайдіть кількість товарів, що розміщені на кожному складі.
+"""
+
+print(f'Total store products = {375291} products\n'
+    f'First and second stocks = {250449} products\n'
+    f'Second and third stocks = {222950} products\n'
+    f'Task solution:\n'
+    f'First stock = {375291 - 222950} products\n'
+    f'Third stock = {375291 - 250449} products\n'
+    f'Second stock = {375291 - (375291 - 250449) - (375291 - 222950)} products')
```

Comment:
And here you are using \n in the end of each row - that is correct 

And in general very good output, it's easy to read 👍